projects/01-pair-matching/hadi-mobarra/webpack/src/styles.css

```diff
@@ -0,0 +1,35 @@
+
+@mixin flexStyle {
```

Comment:
According to your webpack.config.js, this file's extension must be .scss or sass for the sass-loader to load it.
Which means right now it's only using css-loader and style-loader to process this code.... not sure why the scss codes are working.
However I think it's better to change the file name's extension to .scss, if you agree.